feeds/backend.php

```diff
@@ -2,10 +2,13 @@
 $relPath="./../pinc/";
 include_once($relPath.'base.inc');
 include_once($relPath.'misc.inc'); // xmlencode()
+include_once($relPath.'site_vars.php'); // $testing
```

Comment:
`site_vars.php` is pulled in with `base.inc` - you should never need to pull this into a script.

Reply:
Good to know, removed in ee6e168b6c62f5c2edbf44e1bb7e4e8d2890f4cf :)